<HTML>
<HEAD>
<meta charset="UTF-8">
<title>Callback.onFailure - okhttp</title>
<link rel="stylesheet" href="../../../style.css">
</HEAD>
<BODY>
<a href="../../index.html">okhttp</a>&nbsp;/&nbsp;<a href="../index.html">okhttp3</a>&nbsp;/&nbsp;<a href="index.html">Callback</a>&nbsp;/&nbsp;<a href="./on-failure.html">onFailure</a><br/>
<br/>
<h1>onFailure</h1>
<a name="okhttp3.Callback$onFailure(okhttp3.Call, java.io.IOException)"></a>
<code><span class="keyword">abstract</span> <span class="keyword">fun </span><span class="identifier">onFailure</span><span class="symbol">(</span><span class="identifier" id="okhttp3.Callback$onFailure(okhttp3.Call, java.io.IOException)/call">call</span><span class="symbol">:</span>&nbsp;<a href="../-call/index.html"><span class="identifier">Call</span></a><span class="symbol">, </span><span class="identifier" id="okhttp3.Callback$onFailure(okhttp3.Call, java.io.IOException)/e">e</span><span class="symbol">:</span>&nbsp;<a href="https://docs.oracle.com/javase/8/docs/api/java/io/IOException.html"><span class="identifier">IOException</span></a><span class="symbol">)</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-unit/index.html"><span class="identifier">Unit</span></a></code>
<p>Called when the request could not be executed due to cancellation, a connectivity problem or
timeout. Because networks can fail during an exchange, it is possible that the remote server
accepted the request before the failure.</p>
</BODY>
</HTML>
